crates/bevy_app/src/app.rs

```diff
@@ -455,6 +462,36 @@ impl App {
         self
     }
 
+    /// Returns `true` if the [`Resource`] exists in the app world.
```

Comment:
I don't think this method should exist, and even if it should it shouldn't be part of this PR. Simply calling self.main().contains_resource is fine.

Reply:
I added these to simplify many plugins `ready` methods, but generally speaking I think these make the life of plugin builders way easier, instead of accessing the world directly. 

Out of curiosity 😊 , why do you think it shouldn't exist? aren't these equivalent to the other insert/init methods?

Reply:
I created [this](https://github.com/bevyengine/bevy/pull/13425/files), but I'm refactoring method calls not to be dependent on this PR.